templates/header.html

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+<div class="dropdown">
+  <button class="btn btn-default dropdown-toggle pull-right" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+    username@uchicago.edu
+    <span class="caret"></span>
+  </button>
+  <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+    <li><a href="#">Manage Listings</a></li>
+    <li><a href="#"> Verify Profile </a></li>
+    <li><a href="#"> Logout of all instances </a></li>
+  </ul>
+</div>
+</div>
+<div class="row">
+<h1 class="title"> Marketplace </h1>
+</div>
+<img src = "https://upload.wikimedia.org/wikipedia/en/thumb/8/8a/University_of_Chicago_Modern_Etched_Seal_1.svg/1024px-University_of_Chicago_Modern_Etched_Seal_1.svg.png"
```

Comment:
Please de-space the equals sign. Also, please use the UChicago [branding logos][1] and use PNGs instead of SVG.

[1]: http://communications.uchicago.edu/identity/logo